app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitsTicketTypesApiController.php

```diff
@@ -255,6 +260,35 @@ public function getAllBySummitCSV($summit_id){
         }
     }
 
+    /**
+     * @param $summit_id
+     * @return mixed
+     */
+    public function getAllowedBySummit($summit_id){
+        return $this->processRequest(function () use ($summit_id) {
+            $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->resource_server_context)->find($summit_id);
+            if (is_null($summit)) return $this->error404();
+
+            $member = $this->resource_server_context->getCurrentUser();
+            if(is_null($member)) return $this->error404();
+
+            $ticket_types = $this->ticket_type_service->getAllowedTicketTypes($summit, $member->getEmail());
+
+            $resp = new PagingResponse(count($ticket_types), count($ticket_types), 1, 1, $ticket_types);
+
+            return $this->ok
+            (
+                $resp->toArray
+                (
+                    Request::input('expand', ''),
```

Comment:
use static methods to get expand, fields and relations here

Reply:
done